crates/sui-framework/packages/move-stdlib/sources/macros.move

```diff
@@ -243,3 +243,80 @@ public macro fun uq_int_div<$T, $U>(
     if (quotient > $max_t as $U) $abort_overflow;
     quotient as $T
 }
+
+/// Performs an in-place insertion sort on the vector `v` using the comparison function `le`.
+/// The sort is stable, meaning that equal elements will maintain their relative order.
+///
+/// Insertion sort is efficient for small vector, and can be faster than merge sort for almost
+/// sorted vectors (e.g. when the vector is already sorted or nearly sorted).
+public macro fun insertion_sort_by<$T>($v: &mut vector<$T>, $le: |&$T, &$T| -> bool) {
```

Comment:
I'm confused, why is this in `std::macros`. 

If it was up to me this module wouldn't exist ;) But it has to so I can easily reuse macros across shared types. 
The input here is also `vector` so this could just go in `std::vector`?

Reply:
Let'ms move it to vector then. Not a fan, but that was my interpretation of previous discussions.